### ISSUE TITLE
Describe organization

### ISSUE THREAD

**Issue by acka47** [hbz/oerworldmap#29]
As a OER activist I want to add an institution that is somehow involved with OER to the OER World Map.

**Comment by literarymachine** [comment]
Acceptance criteria:

- [ ] name can be entered
- [ ] address can be entered
- [ ] geo coordinates can be entered

**Comment by literarymachine** [comment]
Are these acceptance criteria correct? This is actually up to you, @trugwaldsaenger.

**Comment by acka47** [comment]
I added two acceptance criteria to correspond with https://docs.google.com/spreadsheets/d/1MUA2jg15AlQTwN-M8h52OHGT8Xo5Zf1jHut4zuMQsQI/edit#gid=396085688.

**Comment by acka47** [comment]
@trugwaldsaenger What do you think? Are these acceptance criteria sufficient for the start?

**Comment by literarymachine** [comment]
What do you think, @ack47, should we use http://schema.org/alternateName for abbreviations? Other than that, within http://schema.org/Organization we have http://schema.org/name and http://schema.org/legalName (currently being used as the main name of the entity) available.

**Comment by acka47** [comment]
Yes, I think alternateName is the best choice for now, see also https://docs.google.com/spreadsheets/d/1MUA2jg15AlQTwN-M8h52OHGT8Xo5Zf1jHut4zuMQsQI/edit#gid=396085688. 

(We might start a list of potential schema.org proposals. I think an `abbreviation` property might be a good candidate...)

**Comment by acka47** [comment]
> (We might start a list of potential schema.org proposals. I think an `abbreviation` property might be a good candidate...)

I created a wiki page for this, see https://github.com/hbz/oerworldmap/wiki/Possible-additions-to-schema.org.

**Comment by acka47** [comment]
We got some feedback from Neil Butcher on the metadata scheme I drafted in the linked above google spreadsheet. Regarding the element `streetAddress` he asks:

> Is this needed if you have latitude and longitude? Or is the intention to calculate longitude and latitude from the street address?

I guess we won't expect people to enter the geo coordinates by themselves and should, thus, do the geocding based on the street address. As it looks different in the acceptance criteria and as @philboeselager couldn't tell for sure how this will be implemented, we should get some clarity on this.
